index.html

```diff
@@ -22,7 +22,7 @@
                         My Logo
                     </a>
                     <a href="#">
-                        <img id="hamburger" src="./images/hamburger.svg" alt="menu-icon">
+                        <img id="hamburger" src="./images/hamburger.svg" alt="A menu icon">
                     </a>
```

Comment:
- [ ] As part of the interactivity part of web accessibility, It's possible to navigate through the page, focus on its elements, and select and navigate through its links. Right now, your hamburger is dormant with no interaction. For a better user experience and to maintain industry standards for accessibility, kindly adjust this to keep up with your brillant job so far 👍

Reply:
Hi @Shedrack-Sunday, could you give us an example of that mentioned interaction on the hamburger icon? I'd really appreciate it in order to complete this task. Thanks in advance buddy.

Reply:
Done it @Shedrack-Sunday, I've added an interaction to the hamburger icon.